relay-sampling/src/lib.rs

```diff
@@ -2770,6 +2758,100 @@ mod tests {
         assert_trace_match!(result, 0.03, dsc, 5, 6);
     }
 
+    #[test]
+    /// Tests that the multi-matching works for a mixture of trace and transaction
+    /// 1. Rule is healthcheck bias
+    /// 2. Rule is dev env bias
+    /// 3. Tx rebasing rule part a
+    /// 4. Tx rebasing rule part b
+    /// 5. Factor bias ( to fix rate )
+    /// 6. Uniform sample rate
+    fn test_match_against_rules_with_multiple_event_types_and_matches() {
+        let healthcheck_rule_id = 1002;
```

Comment:
This `healthcheck_rule_id` var is used in one place, it's fine to hardcode this number in the rule id below (especially in tests).

Reply:
@olksdr thanks! fixed in 82c7da46c